<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite name="Failed suite [Default suite]" guice-stage="DEVELOPMENT">
  <parameter name="UserName" value="param-val-not-found"/>
  <parameter name="Name" value="param-val-not-found"/>
  <parameter name="Password" value="param-val-not-found"/>
  <test thread-count="5" name="Default test(failed)">
    <classes>
      <class name="TestngAsserts.AssertionEx2">
        <methods>
          <include name="test10"/>
          <include name="test1"/>
          <include name="test7"/>
          <include name="test4"/>
          <include name="test2"/>
          <include name="test8"/>
        </methods>
      </class> <!-- TestngAsserts.AssertionEx2 -->
      <class name="SkipTest.SkipTest">
        <methods>
          <include name="skipTest"/>
        </methods>
      </class> <!-- SkipTest.SkipTest -->
      <class name="TestngListeners.MyTest">
        <methods>
          <include name="skippingTest"/>
          <include name="failingTest"/>
        </methods>
      </class> <!-- TestngListeners.MyTest -->
      <class name="TestngListeners.SampleTestCase2">
        <methods>
          <include name="test2"/>
        </methods>
      </class> <!-- TestngListeners.SampleTestCase2 -->
    </classes>
  </test> <!-- Default test(failed) -->
</suite> <!-- Failed suite [Default suite] -->
